src/NuGet.Clients/NuGet.Indexing/NuGet.Indexing.csproj

```diff
@@ -18,6 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="Lucene.Net" />
+    <PackageReference Include="SharpZipLib" /> <!-- dependency of Lucene.net. Can delete when Lucene.net has an update with a newer dependency on sharpziplib -->
```

Comment:
Is it ok this one doesn't have `ExcludeAssets="all"`?

Reply:
The `PackageReference` for Lucene.NET doesn't, whereas in `NuGet.VisualStudio.Client` it does.